minichain/p2p.py

```
@@ -1,390 +1,176 @@
+        
+        streams = []
+
+        async def stream_handler(stream):
+            streams.append(stream)
+            peer_id = stream.muxed_conn.peer_id
+            addr = f"peer:{peer_id}"
+            self._to_asyncio.put(("PEER_CONNECTED", None))
+            try:
+                while True:
+                    data = await stream.read(4096)
+                    if not data: break
```

Ruff (0.15.17)

[error] Multiple statements on one line (colon)

(E701)